### PR TITLE
Added initial http2 functionality

### PR DESCRIPTION
**Pull request by chriswilliams13**

### CODE REVIEW

File: ci-build.sh [hunk 1]
```diff
@@ -604,6 +604,34 @@ else
   echo "Testing VERBOSE_ERROR_PAGES works..."
 fi
 
+start_test "Start with http2" "${STD_CMD} \
```
Comment (ghost) [inline ci-build.sh]: could you give this test a more descriptive name? Maybe something like "Test to ensure HTTP/2 is enabled when HTTP2 is set to true

File: ci-build.sh [hunk 2]
```diff
@@ -604,6 +604,34 @@ else
...
+           -e \"HTTP2=TRUE\" \
+           --link \"${MOCKSERVER}:${MOCKSERVER}\" "
+if curl -kv https://${DOCKER_HOST_NAME}:${PORT}/ 2>&1 | grep 'HTTP/2 200' ; then
+  echo "Testing HTTP2 Works"
+else
+  echo "HTTP2 didnt work"
+  exit 1
+fi
+
+start_test "Start with http2" "${STD_CMD} \
```
Comment (ghost) [inline ci-build.sh]: Same here. Something like "Test to ensure HTTP/2 is disabled when HTTP2 is set to false"

Comment (chriswilliams13) [reply ci-build.sh]: I have now addressed this

File: ci-build.sh (hunk 1 above)
Comment (chriswilliams13) [reply ci-build.sh]: I have now addressed this

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**